### PR TITLE
get final commits from Zippy; additions for pagination; remove crosstab jazz

### PR DESCRIPTION
**Pull request by gregtalarico**

### CODE REVIEW

**Comment by vinayakpatil** [comment]
:100:

File: dist/react-datagrid.js [hunk 1]
```diff
@@ -29054,6 +28967,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	          var diff = config.diff[side];
 
 	          newScrollPos = scroll[side] - diff;
+	          console.log(newScrollPos);
```
Comment (gnrlbzik) [inline dist/react-datagrid.js]: you forgot console.log

**Review by gnrlbzik — CHANGES_REQUESTED** [review]

File: dist/react-datagrid.js (hunk 1 above)
Comment (gregtalarico) [reply dist/react-datagrid.js]: let me run the build. maybe we just havent built with that changes

Comment (gregtalarico) [reply dist/react-datagrid.js]: actually i think thats coming from the zippy updates.

**Review by gnrlbzik — APPROVED** [review]

**Pull request merged**